src/test/unit/test_tracer.py

```diff
@@ -341,6 +341,21 @@ def lambda_test_function(event, context):
     ]
 
 
+def test_wrapping_with_auto_tags(context):
+    key = "my_key"
+    value = "my_value"
+
+    @lumigo_tracer(auto_tag=[key])
+    def lambda_test_function(event, context):
+        return "ret_value"
+
+    result = lambda_test_function({key: value}, context)
+    assert result == "ret_value"
+    assert SpansContainer.get_span().function_span[EXECUTION_TAGS_KEY] == [
+        {"key": key, "value": value}
+    ]
+
```

Comment:
Add a test where the value isn't a string and make sure that it behaves as expected.